src/ImageSharp.Drawing/Brushes/ImageBrush{TColor}.cs

```diff
@@ -86,7 +86,7 @@ public ImageBrushApplicator(IImageBase<TColor> image, RectangleF region)
             /// <returns>
             /// The color
             /// </returns>
-            public TColor GetColor(Vector2 point)
+            public override TColor GetColor(Vector2 point)
```

Comment:
I tried a version with a `PopulateRow(..)` style interface it slowed it down filling `SolidBrush`es in the simple benchmarks I tried... I didn't look too hard at it as I figured that could be a follow up to properly investigate the issue.